ui/jstests/test_listing_resources.jsx

```diff
@@ -324,6 +323,212 @@ define(['QUnit', 'jquery', 'listing_resources', 'react',
+            imageDir={listingOptions.imageDir}
+            repoSlug={listingOptions.repoSlug}
+            openResourcePanel={openResourcePanel}
+            updateExportLinkClick={updateExportLinkClick}
+            ref={afterMount}
+            />
+        );
+      }
+    );
+
+    QUnit.test('Assert description expands in ListingResource',
+      function(assert) {
+        var done = assert.async();
+        var showDetail = false;
+
+        var showDescInDetailHandler = function(showDetailFlag) {
+          showDetail = showDetailFlag;
+        };
+
+        var resource = {
+          "run": "2015_Summer",
+          "description": "Lorem Ipsum is simply dummy text of the printing" +
+          "  and typesetting industry. Lorem Ipsum has been the industry's " +
+          " standard dummy text ever since the 1500s, when an unknown " +
+          " took a galley of type and scrambled it to make a type specimen" +
+          " book. It has survived not only five centuries, but also the" +
+          " leap into electronic typesetting, remaining essentially" +
+          " unchanged. It was popularised in the 1960s with the release " +
+          " of Letraset sheets containing Lorem Ipsum passages, and more " +
+          " recently with desktop publishing software like Aldus PageMaker" +
+          " including versions of",
+          "course": "0.001",
+          "xa_avg_grade": 68.0,
+          "lid": 23,
+          "title": "Circuit Schematic Builder",
+          "description_path": "Ops / Content/problem tests / Sample problems" +
+          " / Circuit schematic builder / Circuit Schematic Builder",
+          "xa_nr_views": 9755,
+          "preview_url": "https://www.sandbox.edx.org/courses/DevOps/0.001/" +
+          "2015_Summer/jump_to_id/71101093d5234c8a87488ed9a27db531",
+          "xa_nr_attempts": 717,
+          "resource_type": "problem"
+        };
+
+        var afterMount = function(component) {
+          // Assert link behavior
+          var descriptionDiv = React.findDOMNode(component);
+          var text = $(descriptionDiv).text();
+          assert.equal(text.length, 122);
+
+          var expandLink = $(descriptionDiv).find('a')[0];
+          assert.ok(expandLink, "Expand link exist"); // assert link exist
+
+          React.addons.TestUtils.Simulate.click(expandLink);
+          component.forceUpdate(function() {
+            assert.equal(showDetail,  true, "Description is expanded");
+            done();
+          });
+        };
+
+        React.addons.TestUtils.renderIntoDocument(
+          <DescriptionListingResource
+            resource={resource}
+            showDetail={false}
+            showDescInDetailHandler={showDescInDetailHandler}
+            ref={afterMount}
+            />
+        );
+      }
+    );
+
+    QUnit.test('Assert description collapse in ListingResource',
+      function(assert) {
+        var done = assert.async();
+        var showDetail = false;
```

Comment:
Change this to just `var showDetail;` so that we can tell that `showDetail` changed.